LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) 2021 Vrushti Shah
```

Comment:
@Tejas1510 @Vrushti24 I think, copyright of `Athavani` will be better. Still not sure.